### PR TITLE
(require|disallow)spacesIn*: add more invalid examples and fixes for rul...

### PR DESCRIPTION
**Pull request by hzoo**
...es

#911 @alawatthe 

Fixed the the wrong configuration snippets and then added more invalid snippets for all the rules to be more clear.
Maybe we don't need all those examples?

### CODE REVIEW

File: lib/rules/disallow-spaces-in-function.js [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * #### Example
  *
  * ```js
- * "disallowSpacesInFunctionExpression": {
+ * "disallowSpacesInFunction": {
```
Comment (hzoo) [inline lib/rules/disallow-spaces-in-function.js]: wrong rule name here

File: lib/rules/require-spaces-in-function.js [hunk 2]
```diff
@@ -12,7 +12,7 @@
  * #### Example
  *
  * ```js
- * "requireSpacesInFunctionExpression": {
+ * "requireSpacesInFunction": {
```
Comment (hzoo) [inline lib/rules/require-spaces-in-function.js]: wrong rule name here too

**Comment by hzoo** [comment]
I guess I can just force push again unless you can rerun the build?

**Comment by zxqfox** [comment]
@hzoo I've fixed that bug for 1.11.3, can you try to rebase?

**Comment by zxqfox** [comment]
@hzoo Nice cleanup. ;-)

**Comment by mikesherov** [comment]
@hzoo thanks again as always. Can you please follow the commit message format more closely for future PRs? I almost missed that this fixed #911

**Comment by hzoo** [comment]
Sorry. I've been forgetting haha - more work for you

**Pull request closed without merging**